Editor/GradleWrapperGenerator.cs

```diff
@@ -41,7 +41,7 @@ public void OnPostGenerateGradleAndroidProject(string path)
                 Arguments = $"-jar \"{FindGradleJar()}\" wrapper --gradle-version {gradleVersion} -b {emptyGradleScriptFile}",
                 RedirectStandardError = true,
                 UseShellExecute = false,
-                WindowStyle = ProcessWindowStyle.Hidden,
+                CreateNoWindow = true,
```

Comment:
Oh, nice one, I didn't know of `ProcessStartInfo.CreateNoWindow`.
Seems like a better alternative to `WindowStyle`.